qa/workunits/rbd/cli_generic.sh

```diff
@@ -81,6 +81,21 @@ test_others() {
     rbd snap ls testimg4 | grep -v 'SNAPID' | wc -l | grep 1
     rbd snap ls testimg4 | grep '.*snap1.*'
 
+    # deep copy clone-image
+    rbd snap rm testimg4@snap1
+    rbd rm testimg4
+    rbd rm testimg5
```

Comment:
Nit: the test fails here since ```testimg5``` has a snapshot named ```snap1```

Reply:
Oh sorry, i mistakenly thought `rbd deep copy testimg1 --snap=snap1 testimg5` only copied snapshot.